index.html

```diff
@@ -84,5 +84,224 @@
+          <p class="imgtext marginright">2015</p>
+        </div>
+
+        <p class="cardpara">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <div class="card1-button">
+          <p class="card1-button-paragraph">html</p>
+          <p class="card1-button-paragraph">css</p>
+          <p class="card1-button-paragraph">javascript</p>
+        </div>
+
+        <a href="#" class="card1-project">See Project</a>
+      </div>
+      <!-- End of Card 4 -->
+    </section>
+
+    <!-- End of Work section -->
+
+    <!-- About me section -->
+    <section class="aboutsection">
+      <div class="aboutparagraph">
+        <h2 class="name">About</h2>
+        <h2 class="name">Myself</h2>
+
+        <p class="about">
+          I’m a software developer! I can help you build a product , feature or website Look through
+          some of my work and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.
+        </p>
+
+        <p class="connect">Let's connect</p>
+
+        <div class="vectorimages vimage">
+          <img src="images/introduction/Vector-5.svg" class="vector" alt="vector5" />
+          <img src="images/introduction/Vector-3.svg" alt="vector3" />
+          <img src="images/introduction/Vector-2.svg" alt="vector2" class="vector" />
+          <img src="images/introduction/Vector-1.svg" class="vector" alt="vector1" />
+          <img src="images/introduction/Vector-4.svg" class="vector" alt="vector4" />
+          <img src="images/introduction/Vector.svg" alt="vector" />
+        </div>
+
+        <a href="#" class="card1-project resume">Get my resume</a>
+      </div>
+
+      <div class="skills">
+        <div class="language">
+          <p>Languages</p>
+          <div class="arrow">
+            <i class="fa-solid fa-angle-down"></i>
+          </div>
+        </div>
+
+        <div class="languages">
+          <div class="writtenlanguage">
+            <img src="images/worksection/js.png" alt="javaScript" />
+            <p>JavaScript</p>
+          </div>
+
+          <div class="writtenlanguage">
+            <img src="images/worksection/html.png" alt="HTML" />
+            <p>HTML</p>
+          </div>
+
+          <div class="writtenlanguage">
+            <img src="images/worksection/css.png" alt="CSS" />
+            <p>CSS</p>
+          </div>
+        </div>
+
+        <div class="frameworkdiv">
+          <div class="frameworks">
+            <p>Frameworks</p>
+            <div class="arrow">
+              <i class="fa-solid fa-angle-right"></i>
+            </div>
+          </div>
+          <div class="line"></div>
+
+          <div class="frameworks">
+            <p>Skills</p>
+            <div class="arrow">
+              <i class="fa-solid fa-angle-right"></i>
+            </div>
+          </div>
+
+          <div class="line"></div>
+        </div>
```

Comment:
You got the interpretation of the whole technologies section wrong. Your technologies categories is a list of "Languages", "Frameworks" and "Skills" that should be `li`s grouped in a `ul` tag. then each technology(i.e: Languages)will have it's own subcategory of it own technology in a list (i.e: "JavasScript", "HTML" and "CSS").  Both of these lists should be arranged using flexbox.

Reply:
I will work on all errors immediately. Thank you.